### DISCUSSION TITLE
Increase Max Validator Set to 70

### DISCUSSION THREAD

**Discussion by Atari-buzzk1LL** [fetchai/FIPs#18] [Ideas]
The Fetch-ai Network has remained at a maximum validator set of 60 since it was changed to that via a governance proposal in December of 2021, with [governance proposal #2](https://www.mintscan.io/fetchai/proposals/2). Since then, we have seen many more validators want to join the network and contribute to its decentralization, but we also see that the battle to stay in the active set near the bottom is becoming a larger bar to clear. So the hope is that we can start off a move toward a larger active set by first increasing it to a maximum of 70, and then in roughly 3 months, re-evaluate the set size to accommodate for more increases.

This should hopefully allow for many of the current inactive validators to become active, as well as add a bit of extra room for any newcomers into the space. The other factor that this helps (as previously mentioned) is that it should allow for better decentralization since it will be a broader active set contributing to consensus and will give delegators more options to choose from since there are many amazing validators within the ecosystem.

We’d like to hear the community's input on this, especially validators/potential validators, as the point of these FIP discussions is ultimately to improve the proposals before they become actual FIPs and gov props, so please feel free to make recommendations or give your feedback.

**Comment by SGodshall** [comment]
This sounds like a good proposal to me. Security through decentralisation seems to be a top priority. Is there a document I can read that discusses the relationship between the number of validators securing the network and network transaction speeds?

**Comment by Hash2T** [comment]
PoS has always been critisized for the possibility of it being held hostage in hands of few Staking pools. Now I don't know how many is good enough to make FET properly decentralized. However I would suggest we have a staking plan (similar to minting plan); just a raw idea!
The review of active stake pools every few months also sounds great.
Not a geek or coding nerd, but I can help with a sketchy algorithm. If that helps, count me in

**Comment by mcanvar** [comment]
Sounds fair enough for some validators waiting at the end. As a sufferer of tail validators on other networks I think even after extending the set, organizations should stake some on them for both supporting them and motivating, and also for decentralizing. 💙

**Comment by qf3l3k** [comment]
Increasing number of validators by 10 would bring more opportunity to smaller teams to come to play.
Based on experience from other chains it seems that there is still room to add more validators, so yeah. Let's do it.

**Comment by ardapda** [comment]
As mentioned earlier, this would encourage small validator teams to join the network and let them contribute more. I would definitely support the idea of decentralization and vote yes for this.
Regards AD

**Comment by mawa1** [comment]
This is obviously a very good FIP. One that is much needed for the overall growth of the network, not to mention the added decentralization aspect of it.

**Comment by biganxin** [comment]
From the decentralization and security perspective, we strongly support the expansion. However, judging from the current stake distribution, the two-tier division is rather serious. I think it is necessary to provide delegate support for the lower ranked validators, so as to ensure attracting more active community partners to join, while increasing network fairness and security.

**Comment by juguelio** [comment]
I agree 100% to this proposal. We need higher decentralisation in the network to make more robust

**Comment by social244305-Architect** [comment]
Architect Nodes here. Agree with increasing validator set to give opportunity to new teams. On the decentralization front, top 20 validators control almost 83% of voting power. Achieving decentralization will not happen by just increasing validator set. Somehow delegation needs to be spread out across validators to achieve decentralization and have meaningful value to our voting power.

**Comment by ValidarIOS** [comment]
The proposal is great, but I agree with the other validators who say that the validators at the bottom of the list need help from the project. After all, all new delegators (knowing how POS works) will not want to delegate to those who have a high chance of being inactive, no matter how great a validator he is... A big gap in stakes between validators is a problem of almost all POS, and in my opinion it not overestimated... After all, a small number of validators who are in the first places in almost all networks, by possible prior agreement, can control the price or even blackmail projects! As a solution to the problem, I would consider setting the maximum voice power for the validator at 5-7%. But this is probably a topic for another discussion :)

**Comment by ivandiazperez** [comment]
I see no point in expanding it. The reasons have been explained above already. For many many validators the costs are not even covered now. An expansion of the set will not help to decentralize what's already centralized to a very large extent. It will be just splitting the peanuts at the bottom.
